### PR TITLE
Get background image transition also working for Firefox

### LINKED ISSUE #43: Background transition not working in Firefox
The background transition in the "Discover software by research topic" block is not working in Firefox (Linux, v103.0)

### PR DESCRIPTION
**Pull request by ruester**
# Fix background image transition for Firefox

Fixes #43 

Changes proposed in this pull request:

* use a different approach to change the background of the research field selection, so that it also work for Firefox

How to test:

* build this branch and hover over the research field headings
* the background image of the container should change accordingly to the hovered text
* the background image resets to the default background when leaving the container with the mouse cursor

PR Checklist:

*   [ ] Increase version numbers in `docker-compose.yml`
*   [ ] Link to a GitHub issue
*   [ ] Update documentation
*   [ ] Tests

### CODE REVIEW

File: frontend/styles/custom.css [hunk 1]
```diff
@@ -48,4 +48,35 @@ header input {
...
+}
+
+#research-fields-container #research-fields-list {
+  z-index: 3;
+}
+
+#research-fields-container .research-field-image {
+  z-index: 2;
+  opacity: 0;
+  transition: opacity 0.3s ease 0.1s;
```
Comment (cmeessen) [inline frontend/styles/custom.css:72]:
```suggestion
  transition: opacity 0.3s ease 0s;
```

I remember that the delay in the transition was a fix to solve some kind of flickering or so. To me, the 0.1s feels to long still, and I tried it with 0s. For me it looks good. What do you think?

**Review by cmeessen — APPROVED** [review]
Awesome, thank you :)

File: frontend/styles/custom.css (hunk 1 above)
Comment (ruester) [reply frontend/styles/custom.css:72]: ok, we can remove the delay :+1:

Comment (ruester) [inline frontend/styles/custom.css:72]:
```suggestion
  transition: opacity 0.3s ease;
```

**Pull request merged**